packages/orm/src/drivers/driver.ts

```diff
@@ -0,0 +1,39 @@
+import Knex from "knex";
+import { JoinRow, ManyToManyCollection } from "../collections/ManyToManyCollection";
+import { OneToManyCollection } from "../collections/OneToManyCollection";
+import { OneToOneReference } from "../collections/OneToOneReference";
+import { Entity, EntityConstructor, EntityManager, EntityMetadata } from "../EntityManager";
+import { FilterAndSettings } from "../QueryBuilder";
+import { JoinRowTodo, Todo } from "../Todo";
+
+/** Isolates all SQL calls that joist needs to make to fetch/save data. */
```

Comment:
question: When is `noSQL` support coming?? 😆

Reply:
:-D PRs accepted. :-) 

Fwiw just the basic load / `loadOneToOne` / `loadManyToOne` / etc. methods wouldn't be very hard in NoSQL--it's the queries like `EntityManager.find` that are a PITA. Basically a wire call per nested layer ... ... which no fair pointing out that is technically how Joist works today anyway. :-)